src/model.ts

```diff
@@ -154,31 +155,32 @@ export function convertDescribeJobtoJobDetail(
   // TODO: Convert outputFormats
   return {
     createType: 'Job',
-    jobId: dj.job_id,
-    jobName: dj.name ?? '',
-    inputFile: dj.input_uri,
-    outputPath: dj.output_uri,
-    outputPrefix: dj.output_prefix,
-    environment: dj.runtime_environment_name,
+    jobId: jd.job_id,
+    jobName: jd.name ?? '',
+    inputFile: jd.input_uri,
+    outputPath: jd.output_uri,
+    outputPrefix: jd.output_prefix,
+    environment: jd.runtime_environment_name,
     parameters: jdParameters,
     outputFormats: [],
-    computeType: dj.compute_type,
-    idempotencyToken: dj.idempotency_token,
-    tags: dj.tags,
-    status: dj.status,
-    createTime: dj.create_time,
-    updateTime: dj.update_time,
-    startTime: dj.start_time,
-    endTime: dj.end_time,
+    computeType: jd.compute_type,
+    idempotencyToken: jd.idempotency_token,
+    tags: jd.tags,
+    status: jd.status,
+    statusMessage: jd.status_message,
+    createTime: jd.create_time,
+    updateTime: jd.update_time,
+    startTime: jd.start_time,
+    endTime: jd.end_time,
     scheduleInterval: 'weekday'
   };
 }
 
 export function convertDescribeDefinitiontoDefinition(
-  dj: Scheduler.IDescribeJobDefinition
+  jd: Scheduler.IDescribeJobDefinition
```

Comment:
Again, the `jd` is ambiguous. Perhaps just renaming it to a word like `input` would be easier to understand.

Reply:
Would `describeJob` be too verbose?

Reply:
I don't think they're too verbose

Reply:
inside `convertDescribeDefinitiontoDefinition ` renamed `jd`/`dj` to `describeDefinition`